lib/puma/server.rb

```diff
@@ -648,11 +648,15 @@ def shutting_down?
     # @version 5.0.0
     STAT_METHODS = [:backlog, :running, :pool_capacity, :max_threads, :requests_count].freeze
 
+
```

Comment:
```suggestion
```

Reply:
Note to self - always look at changes before committing...

Fixed